DevelopGitHub/GitHubExtension/GitHubExtension.Statistics.WebApi/Controllers/StatisticController.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System.Security.Claims;
+using System.Threading.Tasks;
+using System.Web.Http;
+using GitHubExtension.Statistics.WebApi.CommunicationModels;
+using GitHubExtension.Statistics.WebApi.Constant;
+using GitHubExtension.Statistics.WebApi.Services.Interfaces;
+using Microsoft.AspNet.Identity;
+
+namespace GitHubExtension.Statistics.WebApi.Controllers
+{
+    public class StatisticController : ApiController
+    {
+        private readonly IGitHubService _gitHubService;
+        private readonly IStatisticsService _statisticsService;
+
+        public StatisticController(IGitHubService gitHubService, IStatisticsService statisticsService1)
+        {
+            this._gitHubService = gitHubService;
+            this._statisticsService = statisticsService1;
+        }
+
+        [Route(RouteConstants.GetUserCommits)]
+        public async Task<IHttpActionResult> GetCommitsForUser()
+        {
+            var claims = User.Identity as ClaimsIdentity;
```

Comment:
The same code which are working with Identity. Please move to this code to some central place to share Identity logic.